server/.env

```diff
@@ -1 +1,4 @@
 TEAM_MEMBERS="Aecio, Shums, Bonnie, Alex, Ryan, Nick"
+
+MONGO_URI="mongodb+srv://pumpkin-spice:hatchways1@peercodereview-qzb4k.mongodb.net/test?retryWrites=true&w=majority"
+MONGO_URI_DEV="mongodb://localhost:27017/pumpkin_spice"
```

Comment:
We are going to want to gitignore this file now, given that these details will be here